hack/.md_links_config.json

```diff
@@ -26,8 +26,15 @@
         },
         {
             "pattern": "https://www.virtualbox.org/*"
+        },
+        {
+            "pattern": "https://avinetworks.com/docs/ako/1.5/handling-objects/*"
         }
     ],
     "retryOn429": false,
-    "aliveStatusCodes": [200, 206, 429]
-}
+    "aliveStatusCodes": [
+        200,
+        206,
+        429
+    ]
+}
```

Comment:
missing new line.

Reply:
please rebase your code to latest main branch, looks like `https://avinetworks.com/*` has been added to this json file.